acapy_agent/did/cheqd/routes.py

```diff
@@ -4,15 +4,85 @@
 
 from aiohttp import web
 from aiohttp_apispec import docs, request_schema, response_schema
-from marshmallow import fields
+from marshmallow import Schema, fields, validate
 
 from ...admin.decorators.auth import tenant_authentication
 from ...admin.request_context import AdminRequestContext
 from ...did.cheqd.cheqd_manager import DidCheqdManager
 from ...messaging.models.openapi import OpenAPISchema
+from ...messaging.valid import CHEQD_DID_EXAMPLE, CHEQD_DID_VALIDATE
 from ...wallet.error import WalletError
 
 
+class VerificationMethodSchema(Schema):
+    """VerificationMethod Schema."""
+
+    id = fields.Str(
+        required=True,
+        metadata={
+            "description": "Verification Method ID",
+            "example": "did:example:123#key-1",
+        },
+    )
+    type = fields.Str(
+        required=True,
+        metadata={
+            "description": "Type of Verification Method",
+            "example": "Ed25519VerificationKey2018",
+        },
+    )
+    publicKeyMultibase = fields.Str(
+        required=True,
+        metadata={"description": "Public Key in multibase format", "example": "z6Mk..."},
+    )
+    controller = fields.Str(
+        required=True,
+        metadata={"description": "Verification controller DID"},
+    )
+
+
+class ServiceSchema(Schema):
+    """Service Schema."""
+
+    id = fields.Str(
+        required=True,
+        metadata={"description": "Service ID", "example": "did:example:123#service-1"},
+    )
+    type = fields.Str(
+        required=True,
+        metadata={"description": "Service Type", "example": "MessagingService"},
+    )
+    serviceEndpoint = fields.List(
+        fields.Str(metadata={"description": "Service endpoint URL"}),
+        required=True,
+        metadata={"description": "Array of Service endpoints"},
+    )
+
+
+class DIDDocumentSchema(Schema):
```

Comment:
Please check if acapy already has a BaseDIDDocumentSchema

as there are more fields like assertionMethod etc. Either we add them here or extend them using Base schema

Reply:
@DaevMithran there is no BaseDIDDocumentSchema or any DocumentSchema